specification/compute/resource-manager/Microsoft.Compute/preview/2019-05-01-preview/ScriptJob.json

```diff
@@ -0,0 +1,625 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RunScripts API",
+    "description": "APIs for managing RunScript jobs.",
+    "version": "2019-05-01-preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/scriptJobs/{jobName}": {
+      "get": {
+        "tags": [
+          "RunScript Job"
+        ],
+        "description": "Retrieves the job identified by the job name.",
+        "externalDocs": {
+          "url": "TBD"
+        },
+        "operationId": "ScriptJob_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/VmNameParameter"
+          },
+          {
+            "$ref": "#/parameters/JobNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Job"
```

Comment:
Usually the convention is to name the return type after the identifier in the URL i.e. `ScriptJob`.

Reply:
Got it; fixed it.